Utilities/File.cpp

```diff
@@ -774,8 +774,9 @@ fs::file::file(const std::string& path, bs_t<open_mode> mode)
 #ifdef _WIN32
 	DWORD access = 0;
 	if (test(mode & fs::read)) access |= GENERIC_READ;
-	if (test(mode & fs::write)) access |= test(mode & fs::append) ? FILE_APPEND_DATA : GENERIC_WRITE;
-
+	if (test(mode & fs::write)) access |= GENERIC_WRITE;
+	if (test(mode & fs::append)) access |= FILE_APPEND_DATA;
```

Comment:
Wrong, this won't work as on Linux.

Reply:
The Linux code farther down does permit the same. On the PS3, the file is created like this and "truncated" (actually enlarged) to 10 MB. Emulator now would do the same.

Reply:
You don't see what's happening: if write is NOT specified, append has no effect, but with your code it starts to "work".

Reply:
Adding GENERIC_WRITE permission kills "atomic append" functionality: it doesn't write at the end of file anymore, but to the current Pointer instead. I'll try to investigate possible solution.